docs/developer/documentation.rst

```diff
@@ -29,9 +29,9 @@ Within the root pymedphys directory run the following command:
 
 .. code:: bash
 
-    sphinx-autobuild docs docs/_build/html
+    sphinx-autobuild -p 7070 docs docs/_build/html
 
-Then within a web browser go to http://127.0.0.1:8000
+Then within a web browser go to http://127.0.0.1:7070
 
```

Comment:
Weirdly enough, `Mosaiq` hosts a webserver on port 8000. Given quite a few of our documentation editors might also be Mosaiq users, I figure it is best to dodge the default port of `sphinx-autobuild`

Reply:
No worries :)